setup.py

```diff
@@ -36,7 +36,7 @@ def run(self):
         'mapentity',
         'cairosvg',
         'cairocffi',
-        'env_file',
+        'python-dotenv',
```

Comment:
**Suggestion:** Consider specifying the minimum required version for `python-dotenv` to ensure compatibility. [best practice]
```suggestion
        'python-dotenv>=1.0.0',
```